### PR TITLE
v1.5 release

### PR DESCRIPTION
**Pull request by bennuttall**
TODO:

- [x] #728 Add pin factory order test
- [x] #721  pinout ASCII art for 3A+/3B+
- [x] #630  Development - mkvirtualenv: command not found
- [x] #495  Some properties missing from docs
- [x] #406  Legal headers
- [x] #379  value property documentation
- [ ] #729 Dave's new `TonalBuzzer` API (done, unmerged)
- [x] Should `JamHat.off()` turn the buzzer off?
- [x] #730 Ensure every class's repr is tested
- [ ] Update changelog (inc. release date)

### CODE REVIEW

**Comment by bennuttall** [comment]
Thanks @waveform80 looking great. I've added a todo list to the top of this PR (which is just a merge request for the docs-tidy-up branch). I think all that's missing now is #630 and your `TonalBuzzer` changes. And are you happy with #728?

**Comment by bennuttall** [comment]
@waveform80 sorry hadn't seen your tones branch, I've created PR #729.

Are you happy with it? I will give it some testing with JamHat. Looks great though!

**Comment by lurch** [comment]
If @waveform80 has time, I think it would be nice if https://github.com/RPi-Distro/python-gpiozero/pull/718#issuecomment-457186244 got fixed for v1.5 too, i.e. add a background thread so that the 'events' fire for InternalDevices.

AIUI if the example shown at https://gpiozero.readthedocs.io/en/stable/api_other.html#gpiozero.TimeOfDay was changed slightly to:
```python
from gpiozero import TimeOfDay, Energenie
from datetime import time
from signal import pause

lamp = Energenie(0)
morning = TimeOfDay(time(7), time(8))

morning.when_activated = lamp.on
morning.when_deactivated = lamp.off

pause()
```
then the lamp wouldn't turn on and off as you might expect it to.

(although I've not written or run any GpioZero code for ages, so if that _does_ work already then please ignore me and accept my apologies!)

**Comment by bennuttall** [comment]
@lurch you are correct, it's still broken. However we are on the brink of a release (probably tomorrow) and I think we agreed to leave it to v1.6. Perhaps if it's easy enough we can fix it and pop into a 1.5.1 release?

**Comment by bennuttall** [comment]
There's an issue with `pi_info()` as it's currently documented.

Both [here](https://gpiozero.readthedocs.io/en/latest/migrating_from_rpigpio.html#pi-information) and [here](https://gpiozero.readthedocs.io/en/latest/api_info.html#gpiozero.PiBoardInfo) we say you can do:

```python
>>> from gpiozero import *
>>> pi_info()
```

but you can't any more. You need to set `Device.pin_factory` first.

Is that a bug that can be fixed in `pi_info()`?

**Comment by waveform80** [comment]
> If @waveform80 has time, I think it would be nice if [#718 (comment)](https://github.com/RPi-Distro/python-gpiozero/pull/718#issuecomment-457186244) got fixed for v1.5 too, i.e. add a background thread so that the 'events' fire for InternalDevices.

Yeah - it would be lovely, but I've already delayed this release enough as it is. I'll try and get that into a 1.5.1 as Ben suggests as it's a major deficiency in the current API.

**Comment by waveform80** [comment]
> ...  You need to set `Device.pin_factory` first.
>
> Is that a bug that can be fixed in `pi_info()`?

Hmmm ... good point. pi_info() should probably be calling _default_pin_factory - although that's a little dirty - I wonder if there's a "cleaner" way ...

**Comment by lurch** [comment]
> we are on the brink of a release (probably tomorrow)

Ahhh, I've not been keeping up with things and didn't realise the release was so imminent :slightly_smiling_face: 
BTW seems there's some issues in the first comment which have been "ticked" but the linked issue is still open and I can't see if it's been implemented? e.g. #406

**Comment by bennuttall** [comment]
#406 is implemented in #727 (the `docs-tidy-up` branch): https://github.com/RPi-Distro/python-gpiozero/pull/727/commits/6f45c90d7dae1e628127911bbe1143f37487e7bd

**Comment by bennuttall** [comment]
Ticked but issue open = done but unmerged

**Comment by lurch** [comment]
P.S. I dunno what's going on with GH not showing the associated code next to my review comments above :-(

File: docs/changelog.rst [hunk 1]
```diff
@@ -32,13 +32,14 @@ Release 1.5.0 (2019-02-01)
   `#634`)
```
Comment (lurch) [inline docs/changelog.rst]: Is there an underscore missing after 634 ?

Comment (bennuttall) [reply docs/changelog.rst]: Looks like it - feel free to push little fixes like that when you see them (if you have time).

**Pull request merged**